sql/core/src/test/resources/sql-tests/inputs/columnresolution-views.sql

```diff
@@ -13,10 +13,8 @@ DROP VIEW view1;
 -- Test scenario with Global Temp view
 CREATE OR REPLACE GLOBAL TEMPORARY VIEW view1 as SELECT 1 as i1;
 SELECT * FROM global_temp.view1;
--- TODO: Support this scenario
 SELECT global_temp.view1.* FROM global_temp.view1;
```

Comment:
I'm not sure if this is a valid case. We should treat `SELECT ... FROM ...` as a 2-levels look up: Level 1, in `FROM` statement, we specify the table name with an optional qualifier(database name). Level 2, in `SELECT` statement, we specify the column name with an optional qualifier(table name).

I don't think it makes sense to specify the database name for column name, the table lookup should be done in the `FROM` statement.